inst/include/distributions/functors/multinomial_lpmf.hpp

```diff
@@ -13,20 +26,28 @@ namespace fims_distributions
     template <typename Type>
     struct MultinomialLPMF : public DensityComponentBase<Type>
     {
-        Type nll = 0.0;
-        fims::Vector<size_t> dims;
-        std::vector<bool> is_na;
+        Type nll = 0.0; /*!< total negative log-likelihood contribution of the distribution */
+        fims::Vector<size_t> dims; /*!< Dimensions of the number of rows and columns of the multivariate dataset */
+        std::vector<bool> is_na; /*!< Boolean; if true, data observation is NA and the likelihood contribution for the entire row is skipped */
         #ifdef TMB_MODEL
-        ::objective_function<Type> *of;
+        ::objective_function<Type> *of; /*!< Pointer to the TMB objective function */
         #endif
-        // data_indicator<tmbutils::vector<Type> , Type> keep;
+        // data_indicator<tmbutils::vector<Type> , Type> keep;  /*!< total negative log-likelihood contribution of the distribution */
```

Comment:
I think this comment should be /*!< Indicator used in TMB one-step-ahead residual calculations */